client/settings/app-settings.jsx

```diff
@@ -55,7 +51,7 @@ class AppForm extends React.Component {
     return (
       <form noValidate={true} onSubmit={onSubmit}>
         <SubmitOnEnter />
-        <VolumeSettings>
+        <FormContainer>
```

Comment:
I had it grouped by divs, using the `FormContainer` but it looked inf of odd. I it row oriented and it looked like this
![row-gap](https://user-images.githubusercontent.com/3045409/123194948-522ee800-d475-11eb-97ba-a68db10c0962.JPG)

Reply:
I think your initial choice to use the `FormControl` component was correct. We want to display all settings in two columns, at least for now. The only tricky thing then becomes how to display the test sound button, since it should be visually connected to the volume slider. One option is either this:

![image](https://user-images.githubusercontent.com/4757944/123345045-7cf06d80-d555-11eb-9bdc-be73322b0ab3.png)

or something like this:

![image](https://user-images.githubusercontent.com/4757944/123345222-e4a6b880-d555-11eb-9bda-9a8b3d53242c.png)

For what it's worth, I think I prefer the former.

Reply:
Also, it's quite annoying that the checkbox label doesn't fit on one line :d. Changing it to something like `Run ShieldBattery on system startup` might help with that.